mcs/tools/wasm-tuner/PInvokeTableGenerator.cs

```diff
@@ -0,0 +1,299 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
From PR comments in https://github.com/dotnet/runtime/pull/38664, we want to stick to the 2 line version. We don't need the third line here, please remove